client/src/game/events/keyboard.ts

```diff
@@ -68,6 +68,12 @@ export function onKeyDown(event: KeyboardEvent): void {
             event.preventDefault();
             event.stopPropagation();
             gameStore.toggleUI();
+        } else if (event.key === "r" && event.ctrlKey) {
+            // Ctrl-r - Re-center/reset the viewport
+            gameStore.setPanX(0);
+            gameStore.setPanY(0);
+            sendClientOptions(gameStore.locationOptions);
+            window.location.reload();
```

Comment:
This unnecessarily reloads the entire page including all the data transfers and bootup logic.
What you want to do instead is tell the canvas to rerender, which you do by telling the various layers that they are no longer valid with: `layerManager.invalidate()`

Reply:
Ok, I've updated the code. CTRL-0 (zero) is now the hot-key and the layermanager invalidated. No more reloads. BTW, if I want to write some settings to the db, how would I approach his? (Think user settings/key settings etc) This might also help with #178. I can have a look at this combination if you want.